firefox-ios/Client/Frontend/Share/TitleSubtitleActivityItemProvider.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UniformTypeIdentifiers
+
+class TitleSubtitleActivityItemProvider: UIActivityItemProvider, @unchecked Sendable {
+    private let message: String
+    private let subtitle: String
+
+    init(_ shareMessage: ShareMessage) {
```

Comment:
nit; Do we need to make the `shareMessage` parameter hidden? I personnally tend to prefer parameters when they are explicit
```suggestion
    init(shareMessage: ShareMessage) {
```